backend/src/core/loaders/swagger.loader.ts

```diff
@@ -37,16 +20,18 @@ const removeCspHeader = (
   next()
 }
 
+const swaggerDocument = YAML.load('../docs/openapi.yaml')
```

Comment:
This path might be dependent on where u're starting the server process from as well so u might want to pay more attention to this

Reply:
Might want to explore `path.resolve`